src/Fantomas/Context.fs

```diff
@@ -589,26 +596,30 @@ let internal printContentBefore triviaNode =
 let internal printContentAfter triviaNode =
     col sepNone triviaNode.ContentAfter printTriviaContent
 
+let rangeStartEq (r1: range) (r2: range) = r1.StartLine = r2.StartLine && r1.StartColumn = r2.StartColumn
```

Comment:
Move to RangeHelpers? Keep the names though, there are better than the ones in RangeHelper ;)

Reply:
done